Modules/_testcapimodule.c

```diff
@@ -281,6 +281,44 @@ test_dict_iteration(PyObject* self, PyObject *Py_UNUSED(ignored))
     Py_RETURN_NONE;
 }
 
+static PyObject *
+pyiter_next(PyObject *self, PyObject *args)
+{
+    PyObject *iter;
+    if (!PyArg_ParseTuple(args, "O:pyiter_next", &iter)) {
+        return NULL;
+    }
+    assert(PyIter_Check(iter) || PyAIter_Check(iter));
```

Comment:
Why PyAIter_Check?

Reply:
It's from the original PR (#105202). I'll tear it out; ditto for the one below. Thanks!